server/sonar-web/src/main/js/components/ui/TagsList.css

```diff
@@ -0,0 +1,29 @@
+.tags-list {
+  padding-left: 6px;
+}
+
+.tags-list i {
+  padding-left: 4px;
+}
+
+.tags-list i::before {
+  font-size: 12px;
+}
+
+.tags-list span {
```

Comment:
Ok I'll remove the `font-size: 11px` in `.tags-list span` but I don't think I can use `.note` for the icon.